roles/openshift_service_catalog/tasks/install.yml

```diff
@@ -72,33 +72,49 @@
     state: list
   register: edit_yaml
 
+# only do this if we don't already have the updated role info
 - name: Generate apply template for clusterrole/edit
   template:
     src: sc_role_patching.j2
     dest: "{{ mktemp.stdout }}/edit_sc_patch.yml"
   vars:
     original_content: "{{ edit_yaml.results.results[0] | to_yaml }}"
+  when:
+  - '"servicecatalog.k8s.io" not in edit_yaml.results.results[0]| to_yaml'
+  - '"settings.k8s.io" not in edit_yaml.results.results[0]| to_yaml'
```

Comment:
these checks seem pretty fragile, especially the quoting bits and the assumption that anything present from these groups means all intended permissions are present. we don't have a great tool for doing covers checks though...